src/jmcomic/jm_plugin.py

```
@@ -1221,23 +1221,86 @@
+                jm_log(client.log_topic(), client.decode(url_to_use))
+            elif is_image:
+                # 图片url
+                client.update_request_with_specify_domain(kwargs, None, is_image)
+
+            resp = request(url_to_use, **kwargs)
+            resp = client.raise_if_resp_should_retry(resp, is_image)
+            return resp
+
+        retry_domain_max_times: int = self.retry_config['retry_domain_max_times']
+        retry_rounds: int = self.retry_config['retry_rounds']
+        for rindex in range(retry_rounds):
```

Ruff (0.12.2)

Loop control variable `rindex` not used within loop body

Rename unused `rindex` to `_rindex`

(B007)